app/config/routes.js

```diff
@@ -18,6 +18,10 @@ export const AUTH_CONFIRM_EMAIL_ROUTE = '/auth/confirmation';
 export const AUTH_RESET_PASSWORD_ROUTE = '/auth/password';
 export const AUTH_RESEND_CONFIRMATION_EMAIL_ROUTE = '/auth/resend';
 export const AUTH_SEND_RESET_PASSWORD_EMAIL_ROUTE = '/auth/reset';
+export const AUTH_SSO_CALLBACK = '/auth/sso';
+export const AUTH_SSO_GOOGLE = '/oauth/google_oauth2';
+export const AUTH_SSO_FACEBOOK = '/oauth/facebook';
+export const AUTH_SSO_UGENT = '/oauth/cas';
```

Comment:
These last three links are handled by the backend if I understand correctly? Then they don't really belong in this list; since they'll only be used in one place I thinks its fine to hardcode them in SigninCard.

Reply:
And now that I think about it, the same goes for `AUTH_SSO_CALLBACK`. Since it will never be used anywhere else, it would be better to just hardcode it in AuthPage so that `config/routes.js` only contains frontend routing.

Reply:
I am convinced that these variables should be put into a separate file and never hardcoded in the code itself. Besides, the SSO login links can also be used from other pages, for example including links to 'sign up with a provider' on the sign up page -- otherwise technically the same thing as 'sign in with a provider'.

Reply:
My main issue is that these routes are handled using a different mechanism from the others, so having them in the same file is confusing if you look at it from the standpoint of someone who is not familiar with the code. I'm fine with keeping them as constants, but at the very least they should be visually separated from the other routes, or, preferably, in a separate file.